src/main/java/com/cburch/logisim/gui/menu/MenuFile.java

```diff
@@ -151,6 +156,8 @@ public void actionPerformed(ActionEvent e) {
       ProjectActions.doSave(proj);
     } else if (src == saveAs && proj != null) {
       ProjectActions.doSaveAs(proj);
+    } else if (src == exportProj && proj != null) {
```

Comment:
Just suggestion - it seems that `proj != null` condition is repeated multiple times. It would be great to simplify all this code by extracting this check first as "umbrella condition" and move all the other code in that block. Kinda "cleanup" side task.

Reply:
Agreed, I just added one, will add a FIXME for refactoring, as it is not a part of this PR.

Reply:
Done anyways :-)